Misc/NEWS.d/next/Library/2024-07-17-12-13-30.gh-issue-120767.BKyogz.rst

```diff
@@ -0,0 +1,4 @@
+Enhance REPL history navigation with partial text support
+
+The REPL now correctly navigates history based on partial text
+in the buffer when using the up arrow.
```

Comment:
```suggestion
Enhance REPL history navigation with partial text support.
The REPL now correctly navigates history based on partial text
in the buffer when using the up arrow.
```

Reply:
I'd be careful with the word *correctly* here, as it could mean many different things.